VideoLocker/res/layout/discussion_responses_response_row.xml

```diff
@@ -24,7 +24,7 @@
 
             <org.edx.mobile.view.custom.ETextView
                 android:id="@+id/discussion_responses_comment_body_text_view"
-                android:layout_width="match_parent"
+                android:layout_width="wrap_content"
```

Comment:
It's a question of layout direction vs. text direction. By using `wrap_content` short pieces of text will appear to be layed out appropriately, while long pieces of text will still be appropriately justified according to their content.